test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -291,6 +292,226 @@ public async Task InstallAndUpdatePackageWithSourceParameterWarnsAsync(ProjectTe
             }
         }
 
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetPackagesConfigTemplates))]
```

Comment:
How many templates is this? I don't think we need more than 1 do we?

Reply:
It's just 1, so we're good.

https://github.com/NuGet/NuGet.Client/blob/63b8dbad9399c4eaade9a2ccdecee7056a69bc7a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs#L311-L314